django/core/management/commands/loaddata.py

```diff
@@ -292,6 +301,12 @@ def parse_name(self, fixture_name):
 
         return name, ser_fmt, cmp_fmt
 
+    def should_skip(self, object):
```

Comment:
The `loaddata.Command.should_skip` method is there for readability reasons, basically we give a "name" to the not-so-readable `if` clause. If there's any problems that this code could introduce in the future, please let me know.